File: Data Structure/3.Linked List/print LinkedList.java
// https://workat.tech/problem-solving/practice/print-linked-list
/** This is the ListNode class definition

class ListNode {
	int data;
	ListNode next;

	ListNode(int data) {
		this.data = data;
		this.next = null;
	}
}
**/

class Solution {
	void printLinkedList (ListNode head) {
	    // add your logic here
		ListNode t = head;
		while(t != null){
			
			System.out.print(t.data+" ");
			t = t.next;
		}
	}
}

File: Data Structure/3.Linked List/PrintLinkedList.py
# https://workat.tech/problem-solving/practice/print-linked-list
"""
This is the ListNode class definition

class ListNode:
	def __init__(self, data=0, next=None):
		self.data = data
		self.next = next

"""


class Solution:
	def printLinkedList(self, head: ListNode) -> None:
		# add your logic here
		t  = head
		while(t != None):
			print(t.data,end=" ")
			t = t.next
		


